docs/source/about/app_examples.rst

```diff
@@ -125,24 +125,24 @@ Linux
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-+----------------------+--------------------------------------------------------------+
-| **Field**            | **Value**                                                    |
-+----------------------+--------------------------------------------------------------+
-| Command Preparations | Do: ``xrandr --output HDMI-1 --mode 1920x1080 --rate 60``    |
-|                      +--------------------------------------------------------------+
-|                      | Undo: ``xrandr --output HDMI-1 --mode 3840×2160 --rate 120`` |
-+----------------------+--------------------------------------------------------------+
++----------------------+-------------------------------------------------------------------------------------------------------------------+
+| **Field**            | **Value**                                                                                                         |
++----------------------+-------------------------------------------------------------------------------------------------------------------+
+| Command Preparations | Do: ``sh -c "xrandr --output HDMI-1 --mode \"${SUNSHINE_CLIENT_WIDTH}x${SUNSHINE_CLIENT_HEIGHT}\" --range 60"``   |
+|                      +-------------------------------------------------------------------------------------------------------------------+
+|                      | Undo: ``xrandr --output HDMI-1 --mode 3840x2160 --rate 120``                                                      |
++----------------------+-------------------------------------------------------------------------------------------------------------------+
 
 Changing Resolution and Refresh Rate (Linux - Wayland)
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-+----------------------+-------------------------------------------------------------+
-| **Field**            | **Value**                                                   |
-+----------------------+-------------------------------------------------------------+
-| Command Preparations | Do: ``wlr-xrandr --output HDMI-1 --mode 1920x1080@60Hz``    |
-|                      +-------------------------------------------------------------+
-|                      | Undo: ``wlr-xrandr --output HDMI-1 --mode 3840×2160@120Hz`` |
-+----------------------+-------------------------------------------------------------+
++----------------------+-----------------------------------------------------------------------------------------------------------------+
+| **Field**            | **Value**                                                                                                       |
++----------------------+-----------------------------------------------------------------------------------------------------------------+
+| Command Preparations | Do: ``sh -c "wlr-xrandr --output HDMI-1 --mode \"${SUNSHINE_CLIENT_WIDTH}x${SUNSHINE_CLIENT_HEIGHT}@60Hz\""``   |
```

Comment:
There should be one more of these Linux examples once you update your branch, from #1448